### PR TITLE
Create source packages for command line utils

### PR DESCRIPTION
**Pull request by natemcmaster**
We want to create "ef.exe" that can be copied w/o needing to also copy "Microsoft.Extensions.CommandLineUtils.dll" along with it.

cc @muratg @pranavkm

### CODE REVIEW

**Comment by muratg** [comment]
I think this is fine. 

cc @Eilon in case he has any objections.

File: NuGetPackageVerifier.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "adx-nonshipping": {
         "rules": [],
         "packages": {
+            "Microsoft.Extensions.CommandLineUtils.Sources": { },
```
Comment (Eilon) [inline NuGetPackageVerifier.json]: Sort 😉

**Comment by Eilon** [comment]
I don't mind having this, but I do feel that this is most likely just going to be at best a temporary solution because there will eventually end up being some other dependency that you won't be able to compile in (or, at least, not as easily). So, fine doing this now, but I'm not sure how long you'll be able to have an `ef.exe` that's standalone copyable.

**Comment by Eilon** [comment]
:shipit:

**Comment by natemcmaster** [comment]
Sorted :)

> I'm not sure how long you'll be able to have an ef.exe that's standalone copyable.

At least until the next release :trollface:. In theory, we'll avoid adding dependencies in ef.exe as much as possible.

**Pull request merged**